tests/phpunit/integration/Core/Conversion_Tracking/Conversion_Event_Providers/PopupMakerTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @package   Google\Site_Kit
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Tests\Core\Conversion_Tracking\Conversion_Event_Providers;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Script;
+use Google\Site_Kit\Core\Conversion_Tracking\Conversion_Event_Providers\PopupMaker;
+use Google\Site_Kit\Tests\TestCase;
+
+class PopupMakerTest extends TestCase {
+
+
+	/**
+	 * PopupMaker instance.
+	 *
+	 * @var PopupMaker
+	 */
+	private $popupmaker;
+
+	public function set_up() {
+		parent::set_up();
+		$this->popupmaker = new PopupMaker( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+	}
+
+	public function test_is_active() {
+		$this->assertFalse( $this->popupmaker->is_active() );
+		define( 'POPMAKE_VERSION', 1 );
```

Comment:
To make sure this constant doesn't effect other tests, can you reset this value when these tests have finished, [as in this ticket](https://github.com/google/site-kit-wp/pull/8651/files#diff-cabdfb1bab040f2727b6159066a59c6d674bc9eb26bd3446b47819c8dd72ce9dR30-R38)